08-transport.Rmd

```diff
@@ -28,36 +30,41 @@ It is therefore unsurprising that transport researchers have long turned to geoc
-Beyond geographic levels and agents, the basic unit of analysis in most transport models is the **trip**, a single purpose journey from an groin 'A' to a destination 'B' [@hollander_transport_2016].
+Beyond geographic levels and agents, the basic unit of analysis in most transport models is the **trip**, a single purpose journey from an origin 'A' to a destination 'B' [@hollander_transport_2016].
 Trips join-up the different levels of transport systems: they are usually represented as *desire lines* connecting *zone* centroids (*nodes*), they can be allocated onto the *route network* as *routes*, and are made by people who can be represented as *agents*.
 
 Another complication is time.
-Although many trips are regular and predictable --- such as the daily commute to work --- transport systems are dynamic and constantly evolving at many levels of temporal resolution.
+Although many trips are regular and predictable --- such as the daily commute to work --- transport systems are dynamic and constantly evolving over time.
 The purpose of geographic transport modeling can be interpreted as simplifying this complexity in a way that captures the essence of transport problems.
 Selecting an appropriate level of geographic analysis can help simplify this complexity, to capture the essence of a transport system without loosing its most important features and variables [@hollander_transport_2016].
 
-Typically models are designed to solve a particular problem.
+Typically, models are designed to solve a particular problem.
 For this reason this chapter is based around a policy scenario that asks:
 how to increase walking and cycling in the city of Bristol?
+Both policies aim to prevent traffic jams, reduce carbon emissions, and promote a healthier life style all of which makes the city greener and thus more attractive and enjoyable to live in.
 The case study city is a coastal settlement in the West of England, described in the next section.
 
 <!-- Idea: make it about reducing CO2 emissions instead. Thoughts? + Multi-model - more complex -->
 
 ## A case study of Bristol {#bris-case}
 
-The case study used for this chapter is a city on the west of England, around 30 km East of the Welsh capital Cardiff.
-An overview of the region's transport network is illustrated in Figure \@ref(fig:bristol), which shows a diversity of transport infrastructure, for cycling, public transport, and private motor vehicle.
+The case study used for this chapter is located in Bristol - a city in the west of England, around 30 km East of the Welsh capital Cardiff.
+An overview of the region's transport network is illustrated in Figure \@ref(fig:bristol), which shows a diversity of transport infrastructure, for cycling, public transport, and private motor vehicles.
 
+<!-- there should be a legend for the study borders (blue lines) as well. I would also reduce the length of the caption since it is basically repeating the content of the legend. -->
```

Comment:
there should be a legend for the study borders (blue lines) as well. I would also reduce the length of the caption since it is basically repeating the content of the legend

Reply:
I disagree that the borders should have a caption: it's cleary which is the inner city boundary and adding another legend would clutter the map. I agree with the comment about caption length and have reduced it accordingly.

Reply:
ok